src/containers/About/About.test.js

```diff
@@ -90,3 +95,22 @@ describe('About', () => {
     ).toBeTruthy();
   });
 });
+
+it('should not display TiggersVersion when value is Not Installed', async () => {
```

Comment:
Not Installed is now gone so needs updating I assume

Reply:
yep updated now